.gitignore

```diff
@@ -0,0 +1 @@
+/target
```

Comment:
other PR's add more files to this (*.class and something else). lets make sure we merge those into this before merging to master so that we don't lose all those changes.